_data/documentsupdated.yaml

```diff
@@ -935,42 +935,7 @@
             - subpagename: Agent Activity
             - subpagename: Current Queue State
             - subpagename: SLA Histogram
-    - documentname: Outbound Reporting API
```

Comment:
Can you confirm this is intended? This removes a number of pages in the nav that don’t seem to be related to ABC, and whose pages don’t seem to be deleted and redirected, either.

Reply:
This isn't a change that I made -- it was merged in from an existing change